lib/config/plugins.js

```diff
@@ -108,6 +108,16 @@ module.exports = {
             pluginNameWithoutPrefix = removePrefix(pluginNameWithoutNamespace);
         let plugin = null;
 
+        if (pluginName.indexOf(" ") >= 0) {
```

Comment:
This should be a regex check because there could also be a tab character or newline character, for example.